tests/unit/conftest.py

```diff
@@ -281,6 +282,10 @@ def application_instance(db_session):
         provisioning=True,
         settings=ApplicationSettings({}),
     )
+
+    application_instance.settings.set("canvas", "sections_enabled", True)
```

Comment:
These were in the test service definition. I reckon the make more sense here, in the instance.